aiohttp/client_ws.py

```diff
@@ -278,7 +298,7 @@ async def receive(self, timeout: Optional[float] = None) -> WSMessage:
                 await self.close()
                 return WSMessage(WSMsgType.CLOSED, None, None)
             except ClientError:
```

Comment:
It looks like `ClientError` is unreachable since `self._reader.read()` will only throw `EofStream` or `WebSocketError`

Reply:
removed in e622cde79265bb51e5c0272a059f490fddcb1b43

Reply:
Maybe it can raise `ServerDisconnectedError` when the connection is lost

Reply:
Looks so. reverted, and added coverage.